hal/src/main/native/sim/SystemServer.cpp

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "hal/SystemServer.h"
+
+extern "C" {
+
+NT_Inst HAL_GetSystemServerHandle(void) {
+  return 0;
```

Comment:
This is actually an interesting future conversation... but yes, makes sense for now.

Reply:
Yes, we do need to have that conversation at some point. But for now we just need something working slightly better.

The main reason I'm adding this is I'm probably going to put the IMU code at the high level, purely because I don't want to make the HAL depend on wpimath, and the IMU gives values in a Quaternion.